webrtc.html

```diff
@@ -11224,6 +11248,11 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
                 attribute is set to the line number in the SDP where the syntax
                 error was detected.</td>
               </tr>
+              <tr>
+                <td><dfn><code>hardware-encoder-error</code></dfn></td>
+                <td>The hardware encoder resources required for the requested
+                operation are not available.</td>
```

Comment:
"... or the hardware encoder doesn't support the provided set of parameters"? Those could feasibly be two separate errors.

Reply:
Raised a separate issue for this question #1602